src/components/DragDrop/DragDrop.stories.tsx

```diff
@@ -0,0 +1,292 @@
+import { Story, StoryObj, Meta } from '@storybook/react';
+import React, { ComponentProps } from 'react';
+import { DragDrop, Props } from './DragDrop';
+import { Default as Card } from '../Card/Card.stories';
+import Heading from '../Heading';
+import MediaBlock from '../MediaBlock';
```

Comment:
FYI a couple of components were moved to `src/upcoming-components/` in https://github.com/chanzuckerberg/edu-design-system/pull/1047.